scylla_elasticsearch_integration/insert_data_into_scylla+elastic.py

```diff
@@ -5,97 +5,91 @@
 ### Using elasticsearch-py ###
 import csv
 from cassandra.cluster import Cluster
+from cassandra import ConsistencyLevel
```

Comment:
I think mostly we can change this examples to use the scylla driver instead.

@DanielHe4rt wdyt?

Reply:
I first tested his sample with the scylla python driver and it worked perfectly.